src/button-dropdown/styles.scss

```diff
@@ -88,4 +88,9 @@ $dropdown-trigger-icon-offset: 2px;
 }
 .dropdown-trigger {
   display: contents;
+
+  & > .trigger-active,
+  & > .trigger-active:hover {
+    color: awsui.$color-text-layout-toggle-active;
+  }
```

Comment:
This also looks like some for of CSS overrides. Need to find an alternative solution

Reply:
Based on the styles of the button dropdown, I couldn't find another way to make the trigger colors match the rest of the buttons (be white). We need to access the button dropdown <button> itself, and this is the only way to do that. What alternative do you see?

Reply:
The other solution I see is adding an internal "type" to the ButtonDropdown itself, which adds the proper styles.

Reply:
For this thread I need to check out this package locally and inspect more closely.

I will try to find some time for it on Monday